app/javascript/components/patient/Patient.js

```diff
@@ -50,6 +50,30 @@ class Patient extends React.Component {
     return <span>{raceArray.length === 0 ? '--' : raceArray.join(', ')}</span>;
   };
 
+  renderEditLink(section, index) {
```

Comment:
Add method comment that also describes params.

Reply:
done